.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,22 @@
+name: CheckStyle
+
+on:
+  pull_request:
+
+jobs:
+  checkstyle:
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
```

Comment:
I also just learned how to skip pointless checks like running CheckStyle on a Dependabot PR.

Reply:
What if it upgrades checkstyle?

Reply:
Good point. I could undo this, or add the ability to trigger the workflow manually. WDYT?

Reply:
If it were me, I'd just remove the condition. It should almost never affect you on a dependabot PR anyway, it's not like it's slow/expensive

Reply:
(I should say also it's nice to know you can do this)